Cargo.toml

```
@@ -41,6 +41,7 @@ bincode = { default-features = false, version = "1.0" }
 bytes = { default-features = false, version = "1.0" }
 criterion = { default-features = false, version = "0.5" }
 csv = "1"
+diesel = { default-features = false, features = ["mysql_backend"], version = "2.2.3" }
```

I think this needs to be 
```suggestion
diesel = { default-features = false, features = ["mysql"], version = "2.2.3" }
```

as we use the full backend for testing

---

Whoops, you're right.  Sorry about that!

---

Fixed